### ISSUE TITLE
Refresh Token action url broken

### ISSUE THREAD

**Issue by leevigraham** [verbb/formie#2022]
### Describe the bug

`refreshFromCache` calls a url with two `?` query strings resulting in a 404 (missing action): `/index.php?p=actions/formie/forms/refresh-tokens?form=requestAnUpdate` 

This is due to the string concatenation here:

![image](https://github.com/user-attachments/assets/12098d69-e871-4401-b173-6e4ff885fdda)

![image](https://github.com/user-attachments/assets/a6e957fb-e1b5-488a-8706-52f2a7c0d30e)

![image](https://github.com/user-attachments/assets/b208ba3a-d69e-4c40-9bba-7ac754974be2)

### Steps to reproduce

Render a form and then call `refreshFromCache`

```twig
<div class="fui-form-wrapper">
    {{ craft.formie.renderForm(form) }}
</div>
{% js %}
    document.addEventListener('onFormieInit', (event) => {
        let Formie = event.detail.formie;
        Formie.refreshForCache('{{ form.formId }}');
    });
{% endjs %}
```

### Form settings

- Multi-page form: Yes or No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

### Craft CMS version

5.3.3

### Plugin version

3.0.1

### Multi-site?

NO

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.1"`.

**Comment by engram-design** [comment]
Fixed in [3.0.2](https://github.com/verbb/formie/releases/tag/3.0.2)
